pandas/core/computation/expr.py

```diff
@@ -635,8 +635,9 @@ def visit_Attribute(self, node, **kwargs):
                 # something like datetime.datetime where scope is overridden
                 if isinstance(value, ast.Name) and value.id == attr:
                     return resolved
+                raise
```

Comment:
why is this raise needed?

Reply:
That's the main change and the real point of the PR. The actual error from getting an invalid attribute should be shown, e.g. "AttributeError: module 'pandas' has no attribute 'thing'". Without it it will say "ValueError: Invalid Attribute context Load" which isn't true, Load is the only valid context.